src/utils/status/common.js

```diff
@@ -1 +1,5 @@
 export const NOT_HANDLED = null;
+
+export const STATUS_INFO = 'INFO';
+export const STATUS_SUCCESS = 'SUCCESS';
+export const STATUS_DANGER = 'DANGER';
```

Comment:
What would be considered a danger?

Reply:
@apinnick Thanks!

Ahh, these are not user-facing texts, it's internal constants that are strings.
In this case, it will be shown to the user as one of the "Error" strings.

Reply:
NP